src/components/Contact/Contact.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Contact.css';
+
+function Contact(props) {
+	const data = props.contact;
+	const location = props.contact.location;
```

Comment:
I think using destructuring for `props` will make things somewhat clearer. This way you have a nice list of exactly what props are supported, and also don't have to write `props.__` everywhere in the component.

```suggestion
function Contact({ 
  contact: { 
    email, 
    phone, 
    website, 
    location: { address, postalCode, city, region, countryCode },
  },
}) {
```